tests/software/bbmap/bbduk/main.nf

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BBMAP_BBDUK } from '../../../../software/bbmap/bbduk/main.nf' addParams( options: [ 'args' : 'trimq=10 qtrim=r', 'suffix' : 'trim' ] )
```

Comment:
```suggestion
include { BBMAP_BBDUK } from '../../../../software/bbmap/bbduk/main.nf' addParams( options: [ 'args' : 'trimq=10 qtrim=r', 'suffix' : '.trim' ] )
```

Reply:
Hi @drpatelh,
Thank you so much for the guidance & feedback; I've added the suggested edits so hopefully it should be up to scratch now. Apologies for any silly errors - I'm a bit of an NF noob so very much learning as I go!
                                                                                                                                                                                                      
Thank you and the rest of the NF-core team for all your hard work developing and maintaining this great resource and if there is anything else you need me to change please give m a shout!